internal/impl/io/input_websocket.go

```diff
@@ -176,9 +178,27 @@ func (w *websocketReader) Connect(ctx context.Context) error {
 	}
 
 	if len(w.openMsg) > 0 {
-		if err := client.WriteMessage(openMsgType, w.openMsg); err != nil {
-			return err
+		if openMsgType == websocket.BinaryMessage {
+			if err := client.WriteMessage(openMsgType, w.openMsg); err != nil {
+				return err
+			}
+		} else if openMsgType == websocket.TextMessage {
+			openMessag := string(w.openMsg)
+			messages := strings.Split(openMessag, "\n")
+			for _, msg := range messages {
+				if len(msg) == 0 {
+					continue
+				}
+
+				// TODO: will need remove such log later cause it may contain sensitive info
+				w.log.Infof("sending open message %s to server", msg)
```

Comment:
remove it?

Reply:
I want to keep it before integration with UI is done, as I am not 100% sure if UI can send the right messaage in such case